server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -95,11 +95,17 @@ export function create(
                 content: blobData.content,
                 encoding: "base64",
             };
-            uploadBlob(uri, requestBody).then((data: git.ICreateBlobResponse) => {
-                response.status(200).json(data);
-            }, (err) => {
-                response.status(400).end(err.toString());
-            });
+
+            axios.post(`${externalHistorianUrl}/repos/${tenantId}/git/blobs?token=${requestToken}`, requestBody, {
```

Comment:
Can you just use `uri`, defined above? This might also remove the CodeQL check

Reply:
We're hoping this route can just be eliminated entirely (note the TODO above)

Reply:
I tried using the defined uri as well, didn't seem to work, unfortunately.